odfdom/src/main/java/org/odftoolkit/odfdom/pkg/ZipHelper.java

```diff
@@ -95,7 +95,7 @@ String entriesToMap(Map<String, ZipArchiveEntry> zipEntries) throws IOException,
             }
             // JDK 6 -- the try/catch is workaround for a specific JDK 5 only problem
             if (!e.getMessage().contains("missing entry name")
-                && !System.getProperty("Java.version").equals("1.5.0")) {
+                && !"1.5.0".equals(System.getProperty("Java.version"))) {
```

Comment:
this is dead code anyway, we require, i forgot, certainly at least java 8 - easiest to just delete the whole thing

Reply:
It's not dead, fixed it because I ran into the NPE

Reply:
well if you want to get pedantic about it, there is no dead code but an always-false condition, and i'm arguing that going from {false, ⊥} to {false} is not as much an improvement as just removing the condition.

Reply:
I'm not being pedantic. it's not always false. on my setup System.getProperty("Java.version") returns null raising an NPE

Reply:
The code is not needed anymore, that I can agree on

Reply:
Seems to be an invalid assumption to rely on the system property "Java.version" being set

Reply:
hmm, turns out system properties are not case-insensitive. "java.version" would work, "Java.version" doesn't